01_fib/cuminandpaprika/main_test.go

```diff
@@ -0,0 +1,89 @@
+package main
+
+import (
+	"bytes"
+	"strconv"
+	"testing"
+)
+
+func TestMainCallsFib(t *testing.T) {
+	// Arrange
+	var buf bytes.Buffer
+	out = &buf
+
+	// Act
+	main()
+
+	// Assert
+	expected := strconv.Quote("0\n1\n1\n2\n3\n5\n8\n")
```

Comment:
FWIW: [consider using raw strings](https://github.com/anz-bank/go-course/wiki/Lab-1:-Collected-Code-Review-Comments#consider-using-raw-strings)

Reply:
Wasn't a big fan of using raw strings due to the formatting, it takes up so many lines! 

Would using a test data file be a typical alternative approach for cleaning up the code for future reference?